### PR TITLE
removes react warnings over missing key= and required props

### LINKED ISSUE #281: fix react warnings
currently:

```
warning.js:45 Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `WorkspaceRegionView`. See https://fb.me/react-warning-keys for more information.
```

and a few around blotter re: missing width props expected for cell renderers. 
```
vendor.js?8324738…:1 Warning: Failed propType: Required prop `width` was not specified in `t`. Check the render method of `FixedDataTableCell`.
```

### PR DESCRIPTION
**Pull request by DimitarChristoff**
affects blotter and workspace view, closes #281

### CODE REVIEW

File: src/client/src/ui/blotter/views/blotterView.jsx [hunk 1]
```diff
@@ -60,61 +60,64 @@ export default class BlotterView extends ViewBase {
...
         header={<Cell className='blotter__trade-field--align-right'>Notional</Cell>}
         cell={props => {
           let trade = this.state.model.trades[props.rowIndex];
-          return (<NotionalCell
-            className='blotter__trade-field--emphasized blotter__trade-field--align-right'
-            notionalValue={trade.notional}
-            suffix={' ' + trade.currencyPair.base} />);
+          return (
+            <NotionalCell
+              width={props.width}
```
Comment (KeithWoods) [inline src/client/src/ui/blotter/views/blotterView.jsx]: does notional really need a width, can we remove this functionality?

File: src/client/src/ui/blotter/views/blotterView.jsx [hunk 2]
```diff
@@ -60,61 +60,64 @@ export default class BlotterView extends ViewBase {
...
         header={<Cell>Status</Cell>}
-        cell={props => (<Cell className='blotter__trade-status'>{this.state.model.trades[props.rowIndex].status.name}</Cell>)}
+        cell={props => <Cell className='blotter__trade-status'>{this.state.model.trades[props.rowIndex].status.name}</Cell>}
         flexGrow={1}
         width={80}/>,
       <Column
         key='Value date'
         header={<Cell>Value date</Cell>}
-        cell={props => (<DateCell prefix='SP. ' format='%d %b' dateValue={this.state.model.trades[props.rowIndex].valueDate} />)}
+        cell={props => <DateCell width={props.width} prefix='SP. ' format='%d %b' dateValue={this.state.model.trades[props.rowIndex].valueDate} />}
```
Comment (KeithWoods) [inline src/client/src/ui/blotter/views/blotterView.jsx]: same with this, can we do away with width?

Comment (DimitarChristoff) [inline src/client/src/ui/blotter/views/blotterView.jsx]: no. we can't - both subclass the FDT's `Cell`, which has `width` as a required prop.

Comment (KeithWoods) [inline src/client/src/ui/blotter/views/blotterView.jsx]: ok cool

**Pull request merged**